main.py

```diff
@@ -29,6 +29,25 @@ def send_discord(payload):
 
     return response
 
+
+#fetch whales to mark their trades
+def fetch_whales():
+
+    whaleSize = 100000 #size threshold
+    whaleSize = whaleSize * 1e18 #convert using mantissa
+    global whales
```

Comment:
There's a few ways that might be better to do this instead of a global like this.

You can either pass around `whales` (returning it in this function, then passing it down into `fetch_contract_activity`, or you could define `whales = []` above `watch_for_changes` at the bottom of this file, then it should be accessible globally (in a way that doesn't use `global` - I generally discourage using this because there's a more discoverable way to do it).